cfgov/jinja2/v1/external-site/index.html

```diff
@@ -6,7 +6,10 @@
 {% set request_url = request_copy.pop('ext_url', None) %}
 {% set params = request_copy | urlencode %}
 {% if request_url %}
-   {% set request_url = request_url[0] + '&' + params %}
+   {% set request_url = request_url[0] %}
+   {% if params %}
+      {% set request_url = request_url + '&' + params %}
+   {% endif %}
```

Comment:
Okay. Could be shortened to `{% set request_url = request_url[0] + '&' + params if params else request_url[0] %}`